src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -91,14 +90,19 @@ export const UserProfileModal: React.FC<UserProfileModalProps> = ({
             <div className="profile-extras">
               {venue?.profile_questions?.map((question) => (
```

Comment:
I'm sorry I don't understand what this means. It would be fantastic if no input = allows to save, no input = no display on profile modal!

Reply:
@sofisparkle that's precisely what this feature would allow

Reply:
sorry, read conversation again. the issue now is that all profile questions require an answer. If we make answers optional, it would be possible to not code them in the individual profile dataset, in which case they wouldn't render. Not sure if new issues would arise if we implement differently, but happy for any suggestions. Otherwise, this does seem to do the trick.

Reply:
@margulies As called out somewhere in my initial review (or inferred, or I thought about it but never explicitly wrote it.. honestly not sure at this stage), most of your changes here seem to be related to the rendering side of things, but less so on the validation/backend functions/etc side of things that it may impact.

eg. searching the codebase for `profile_questions`:

- https://github.com/sparkletown/sparkle/search?q=profile_questions

Shows usages all across the codebase, including in validations such as:

- `src/pages/Admin/Venue/DetailsValidationSchema.ts`
- `src/pages/Admin/Details/ValidationSchema.ts`

And backend functions such as:

- `functions/venue.js`

How certain are we that the changes here cover all of those cases correctly, given no changes seem to have been made to those files?

Reply:
> This looks like it is only going to render when the profile_questions item in venue config has elements to render, so I wonder if the changes being made here are even needed? We should just not have them set in the database if they aren't wanted?

Yes, if there is no `profile questions` set, it will not render them at all. It's our logic.

The change here suggests not to render `questions`(which were set on the backend, and are optional), which were not answered, instead of the `THEY WERE NOT ANSWERED` text, which makes sense to me

Reply:
As per 
![image](https://user-images.githubusercontent.com/63194656/120862501-bc98ea80-c591-11eb-8133-a932ffd3780a.png)

my guess was right